tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py36,py37,py38,py39,py311{py27,py36,py37,py38,py39,py311}-pandas
+envlist = py27,py36,py37,py38,py39,{py27,py36,py37,py38,py39,py311}-pandas
```

Comment:
Why was this removed from the list?

Reply:
I just renamed it, because right now it creates an environment called `py311py27-pandas` (`py311py27-pandas create: /home/alessio/crunch/scrunch/.tox/py311py27-pandas`)